src/appengine/handlers/cron/ml_train.py

```diff
@@ -29,17 +35,18 @@ def get(self):
     """Handle a GET request."""
     for job in data_types.Job.query():
 
-      if not utils.string_is_true(
-          job.get_environment().get('USE_CORPUS_FOR_ML')):
+      models = job.get_environment().get('ML_MODELS_TO_USE')
+      if not models:
         continue
 
       task_list = []
-      if utils.string_is_true(job.get_environment().get('USE_GRADIENTFUZZ')):
-        task_list.append('train_gradientfuzz')
-      if utils.string_is_true(job.get_environment().get('USE_RNN_GENERATOR')):
-        task_list.append('train_rnn_generator')
+      for model_name in models.split(','):
```

Comment:
models.strip().split(',')

Reply:
I think the strip below should cover any cases this would handle, and that one will be necessary regardless for cases like "... = model1, model2".